backend/src/assets/404.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Sleep API</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="description"
+      content="Optimize your Pokémon Sleep cooking with Sleep API's simulator. Explore the in-depth cooking tier list and find the best teams for your cooking strategy."
+    />
+    <meta property="og:title" content="Pokémon Sleep Cooking Simulator & Tier List | Sleep API" />
+    <meta
+      property="og:description"
+      content="Optimize your Pokémon Sleep cooking with Sleep API's simulator. Explore the in-depth cooking tier list and find the best teams for your cooking strategy."
+    />
+    <meta property="og:image" content="https://api.sleepapi.net/sleepapi-banner.png" />
+    <meta property="og:url" content="https://sleepapi.net" />
+    <link rel="icon" href="/cook.ico" type="image/x-icon" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="/global.css" />
+    <style>
+      #bannerImage {
+        max-width: 90%;
+        height: auto;
+        margin-left: 5%;
+        margin-right: 5%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container" style="margin-bottom: 50px">
+      <header class="d-flex justify-content-center mb-2 mt-2">
+        <a href="/">
+          <img
+            id="bannerImage"
+            src="sleepapi-banner.png"
+            alt="Sleep API Cooking Simulator Logo"
+            title="Pokémon Sleep Cooking Simulator"
+          />
+        </a>
+      </header>
+      <div class="card-container">
+        <div class="card" style="margin: auto">
+          <div class="card-body">
+            <div class="row justify-content-center mb-2">
+              <h1 style="text-align: center; color: #ddd">Error 404</h1>
+              <h1 style="text-align: center; color: #ddd">Sneasel could not find your page</h1>
+              <h2 style="text-align: center; color: #ddd">
+                <a href="/" class="btn btn-danger btn-navigation">To front page</a>
+              </h2>
+              <img src="cook.gif" alt="Sneasel cooking" />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <footer class="text-center py-2" style="background-color: #555; color: #ddd">
```

Comment:
Footer doesn't stick to bottom of the page, the reason is that app.css isn't imported here.
We kind of want to move away from app.css anyway, it was just a shitty w3schools initial solution like half a year ago. 

Instead you could probably just add this to global.css (which is already imported here):
```css
footer {
  position: absolute;
  left: 0;
  bottom: 0;
  width: 100%;
  background-color: black;
  color: white;
  text-align: center;
}
```